numba/misc/numba_sysinfo.py

```diff
@@ -392,8 +392,7 @@ def get_sysinfo():
                 sys_info[_cu_mvc_available] = False
 
             sys_info[_cu_mvc_needed] = cu_rt_ver > cu_drv_ver
-            mvc_used = hasattr(cudadrv.driver, 'CubinLinker')
-            sys_info[_cu_mvc_in_use] = mvc_used
+            sys_info[_cu_mvc_in_use] = config.CUDA_ENABLE_MINOR_VERSION_COMPATIBILITY
```

Comment:
The flag is an `int`, so it needs converting to `bool` to present consistently with the other CUDA settings reported by the sysinfo tool:

```suggestion
            sys_info[_cu_mvc_in_use] = bool(
                config.CUDA_ENABLE_MINOR_VERSION_COMPATIBILITY)
```

to give e.g.:

```
CUDA Minor Version Compatibility Available    : True
CUDA Minor Version Compatibility Needed       : False
CUDA Minor Version Compatibility In Use       : False
```

as opposed to:

```
CUDA Minor Version Compatibility Available    : True
CUDA Minor Version Compatibility Needed       : False
CUDA Minor Version Compatibility In Use       : 0
```